src/index.js

```diff
@@ -1,21 +1,26 @@
 import './style.css';
+import { fillList } from './modules/fill-render.js';
+import { addTaskEvent, taskDescription, TaskList } from './modules/add-task.js';
+// eslint-disable-next-line import/named
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.